mirror.js

```diff
@@ -13,7 +12,9 @@ async function handleRequest(event) {
   options.cf.image.height = 400;
 
   const imageURL = url.searchParams.get('img');
+  if (!imageURL) return new Response('Missing "img" value', { status: 400 })
 
+  const { pathname } = new URL(imageURL)
```

Comment:
This doesnt seem to be used, can you remove it?

Reply:
Good catch, removed it @bonustrack 👍🏻